src/seb/registered_models/sentence_transformer_models.py

```diff
@@ -524,6 +524,25 @@ def create_xlmroberta_en_da_sv_nb() -> SebModel:
     )
 
 
+@models.register("MeMo-BERT-03")
+def create_memo_bert_03() -> SebModel:
+    hf_name = "MiMe-MeMo/MeMo-BERT-03"
+    meta = ModelMeta(
+        name=hf_name.split("/")[-1],
+        huggingface_name=hf_name,
+        reference=f"https://huggingface.co/{hf_name}",
+        open_source=True,
+        languages=["da"],
```

Comment:
```suggestion
        languages=["da", "nb"],
```
Also norwegian right?

Reply:
I just talked to Alie about this and she said that even the Norwegian authors were publishing in Danish, so we should probably keep it this way.